pkg/controller/traceflow/controller.go

```diff
@@ -237,6 +237,7 @@ func (c *Controller) syncTraceflow(traceflowName string) error {
 		return err
 	}
 	switch tf.Status.Phase {
+	//lint:ignore SA1019 Allow existing use.
 	case "", crdv1alpha1.Pending:
```

Comment:
@luolanzone you can remove the lint directive and the `Pending` reference in your Traceflow graduation PR